apps/angular/src/app/features/anime-table/anime-table.component.ts

```diff
@@ -46,6 +109,138 @@ export class AnimeTableComponent {
 	/** Column key. */
 	protected readonly columnKey = ColumnKey;
 
+	/** Data source. */
+	protected animeList$ = new Observable<readonly Anime[]>();
```

Comment:
Please, rework it the way we've discussed:
```suggestion
	protected readonly animeList$ = this.createAnimeListStream();
```